features/step_definitions/basic_steps.rb

```diff
@@ -83,15 +85,23 @@
 end
 
 Given (/^I fill in the new charity page validly$/) do
-  fill_in 'organisation_address', :with => '64 pinner road'
+  fill_in 'organisation_address', with: '64 pinner road'
+  fill_in 'organisation_description', with: 'This is an awesome charity'
+  fill_in 'organisation_name', with: 'Friendly charity'
+  fill_in 'organisation_postcode', with: 'HA1 4HZ'
+end
+
+Given (/^I fill in the new charity page with an invalid website$/) do
+  fill_in 'organisation_description', :with => 'This is an awesome charity'
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)